example/android/build.gradle

```diff
@@ -20,10 +21,14 @@ buildscript {
 
 allprojects {
     repositories {
+        mavenLocal()
         google()
         mavenCentral()
         // Ravelin - optional - only required if you are using Ravelin
-        maven { url 'https://maven.ravelin.com/public/repositories/core-android/' }
+        maven {
+          url 'https://maven.ravelin.com/public/repositories/core-android/'
+          }
+      maven { url("$rootDir/../node_modules/detox/Detox-android") }
```

Comment:
Can we please add this:
`// This is for Detox test framework`

So we don't forget why this is needed

Reply:
Done